### DISCUSSION TITLE
Configure no trailing slash in URLs?

### DISCUSSION THREAD

**Discussion by adrianandreias** [squidfunk/mkdocs-material#6332] [Q&A]
Is there are a way to configure URL's to have no trailing slash?
I know directories are used and `index.html` is loaded from the directory, but I can make the necessary web server config for this to work.

I use Jekyll in other projects and I can configure no trailing slash in URL's.

Is there a way to do this with MkDocs?

**Comment by alexvoss** [comment]
I am not aware of such an option. There have been a bunch of discussions about the trailing slash over at MkDocs. It seems the trailing slashes used to be necessary for the live reload server to work but [this may have changed](https://github.com/mkdocs/mkdocs/pull/2507)? The server still sends a redirect to the version with a slash at the end, though. So does my own fairly vanilla Nginx install but that should be configurable.

The question remains whether the links produced by MkDocs/Material could be produced without the slash. If you care enough, you could write a [hook](https://www.mkdocs.org/user-guide/configuration/#hooks) that uses the [on_post_page](https://www.mkdocs.org/dev-guide/plugins/#on_post_page) event to rewrite URLs.

Having said this, it strikes me that you did not say what the reason is to get rid of the slashes?

**Comment by adrianandreias** [comment]
> Having said this, it strikes me that you did not say what the reason is to get rid of the slashes?

In my opinion, URL's without the trailing-slash simply look better.
